cri-o/config.json.template

```diff
@@ -0,0 +1,410 @@
+                "rbind",
+                "rprivate",
+                "rw",
+                "mode=755"
+            ],
+            "source": "/root",
+            "type": "bind"
+        },
+        {
+            "destination": "/home",
+            "options": [
+                "rbind",
+                "rprivate",
+                "rw",
+                "mode=755"
+            ],
+            "source": "/home",
+            "type": "bind"
+        },
+        {
+            "destination": "/mnt",
+            "options": [
+                "rbind",
+                "rw",
+                "rprivate",
+                "mode=755"
+            ],
+            "source": "/mnt",
+            "type": "bind"
+        },
+        {
+            "destination": "/run",
+            "options": [
+                "rshared",
+                "rbind",
+                "rw",
+                "mode=755"
+            ],
+            "source": "${RUN_DIRECTORY}",
+            "type": "bind"
+        },
+        {
+            "destination": "/run/systemd",
+            "options": [
+                "rslave",
+                "bind",
+                "rw",
+                "mode=755"
+            ],
+            "source": "/run/systemd",
+            "type": "bind"
+        },
+        {
+            "destination": "/var/run",
+            "source": "/var/run",
+            "options": [
+                "rbind",
+                "rslave",
+                "rw"
+            ],
+            "type": "bind"
+        },
+        {
+            "destination": "/var/log",
+            "options": [
+                "rbind",
+                "rslave",
+                "rw"
+            ],
+            "source": "/var/log",
+            "type": "bind"
+        },
+        {
+            "destination": "/var/lib",
+            "options": [
+                "rbind",
+                "rshared",
+                "rw"
+            ],
+            "source": "${STATE_DIRECTORY}",
+            "type": "bind"
+        },
+        {
+            "destination": "/dev",
+            "options": [
+                "rprivate",
+                "rbind",
+                "rw",
+                "mode=755"
+            ],
+            "source": "/dev",
+            "type": "bind"
+        },
+        {
+            "destination": "/sys",
+            "options": [
+                "rprivate",
+                "rbind",
+                "rw",
+                "mode=755"
+            ],
+            "source": "/sys",
+            "type": "bind"
+        },
+        {
+            "destination": "/proc",
+            "options": [
+                "private"
+            ],
+            "source": "/proc",
+            "type": "proc"
+        },
+        {
+            "destination": "/opt/cni",
+            "options": [
+                "rbind",
+                "rprivate",
+                "rw",
+                "mode=755"
+            ],
+            "source": "/opt/cni",
```

Comment:
@giuseppe I think we can change the cni location via the configuration file, but by default the ``containernetworking-*`` packages put the results in ``/opt/cni``. I'm fine either way.